### DISCUSSION TITLE
plea for prioritized support

### DISCUSSION THREAD

**Discussion by jochen-schuettler** [sustainable-computing-io/kepler#1126] [General]
At [this](https://github.com/sustainable-computing-io/kepler/issues/1100) and [this](https://github.com/sustainable-computing-io/kepler/issues/1109) point I opened issues, and [here](https://github.com/sustainable-computing-io/kepler/discussions/1104) I asked for a feature. All of them are at least a week stale now. Please be aware that we have kepler running on the VM machines of our customer, a big player in upper price fashion. This is kind of a evaluation agreement, we can keep it running until a decision meeting next Jan/Feb. If that decision runs in favor it could be a stepping stone in sustainable computing with them and also for us. Until now the measurement is not helping much, the evaluation would be cancelled then. Please, if you can manage, prioritize my issues and work on them. Thank you!

**Comment by rootfs** [comment]
@jochen-schuettler appreciate your input. At the moment, we are focusing on the feature development and core issue resolution. The grafana dashboard and prometheus integration have many moving parts, it is hard to diagnose, reproduce, or identify root cause without having direct access to the running environment.

**Reply by jochen-schuettler** [reply]
I know. I'd like to offer more direct help in identifying causes. I'm also setting up a test environment that might show the same symptoms and could be shared with you.

**Comment by jochen-schuettler** [comment]
Two of the issues are resolved, the other is minor. So no urgent need for help any more. I'm closing this.

**Comment by rootfs** [comment]
@jochen-schuettler thanks for the update. If you can share your diagnostic and resolution with the community, would you like to contribute to the kepler-doc https://github.com/sustainable-computing-io/kepler-doc
